src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -99,6 +100,22 @@ public static Diagnosis parseDiagnosis(String diagnosis) throws ParseException {
         }
         return new Diagnosis(trimmedDiagnosis);
     }
+    // @@ GAO JIAXIN
+    /**
+     * Parses a {@code String visitor} into a {@code visitor}. Leading
+     * and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException
```

Comment:
Please adhere to the format for Javadocs used throughout the code i.e. there should be no newline between a Javadoc tag and its content.